include/ews/ews.hpp

```diff
@@ -21568,19 +21601,21 @@ class basic_service final
 #endif // 0
 
     //! Retrieves an attachment from the Exchange store
-    attachment get_attachment(const attachment_id& id)
-    {
-        auto response = request("<m:GetAttachment>"
-                                "<m:AttachmentShape>"
-                                "<m:IncludeMimeContent/>"
-                                "<m:BodyType/>"
-                                "<m:FilterHtmlContent/>"
-                                "<m:AdditionalProperties/>"
-                                "</m:AttachmentShape>"
-                                "<m:AttachmentIds>" +
-                                id.to_xml() +
-                                "</m:AttachmentIds>"
-                                "</m:GetAttachment>");
+    attachment get_attachment(const attachment_id& id,
+                              bool include_mime_content = false)
+    {
+        std::stringstream sstr;
+        sstr << "<m:GetAttachment>"
+                "<m:AttachmentShape>"
+                "<t:IncludeMimeContent>"
```

Comment:
I would add a remark here that the namespace prefix `t` is used by intention. Like you explained in your pull request's comment. Otherwise, it might get hard later to understand why here the prefixes `m` and `t` are used.

Reply:
I'm not sure it's necessary to explain it right there.
Most requests include tags from both namespaces. 
`sync_folder_hierarchy_impl(...)` for example has the message FolderShape with the type BaseShape.

From what I understand, it was just a typo in the `get_attachment(...)` function.
And by coincidence, this typo didn't cause any further issues.
Making a specific remark in the code here would raise the question why other functions don't need the remark.

Reply:
Ok! These prefixes can be also found in the examples of the official EWS docs.